config/initializers/bullet.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+return if Rails.env.production?
+
+Rails.application.configure do
+  config.after_initialize do
+    Bullet.enable = true
+    Bullet.bullet_logger = true
+
+    if Rails.env.development?
+      Bullet.alert = true
+      Bullet.console = true
+      Bullet.rails_logger = true
+      Bullet.add_footer = true
+    end
+
+    if Rails.env.test?
+      Bullet.raise = true # raise an error if n+1 query occurs
```

Comment:
```suggestion
      Bullet.raise = true # raise an error if n+1 query occurs
      Bullet.unused_eager_loading_enable = false
```

Because you often make tests that does not contain enough data to merit the `includes` that are needed with a full dataset.